tasks/config.yaml

```diff
@@ -16,11 +16,11 @@
     label: "{{ render_path }}"
   vars:
     template_dir: "{{ role_path }}/templates"
-    render_path: "{{ nginx.prefix.config }}/{{ item|strip_prefix(template_dir + '/')|strip_suffix('.j2') }}"
+    render_path: "{{ nginx.prefix.config }}/{{ item|strip_prefix(template_dir + '/') }}"
   with_fileglob:
-    - "{{ role_path }}/templates/*.j2"
-    - "{{ role_path }}/templates/include/*.j2"
-    - "{{ role_path }}/templates/http.d/*.j2"
+    - "{{ role_path }}/templates/*.conf"
+    - "{{ role_path }}/templates/include/*.conf"
+    - "{{ role_path }}/templates/http.d/*.conf"
```

Comment:
IMO it would be good to remove `with_fileglob` entirely here. File globbing was the reason I started using the `.j2` extension.
```yaml
- name: Template nginx config
  loop:
    - src: nginx.conf
      dest: "{{ nginx.prefix.config }}/nginx.conf"
    - src: http.d/default.conf
      dest: "{{ nginx.prefix.config }}/http.d/default.conf"
    # ... more files
  loop_control:
    label: "{{ item.dest }}"
  template:
    src: "{{ item.src }}"
    dest: "{{ item.dest }}"
```

Reply:
yep I could name the files explicitly